app/components/memorable_date_component.html.erb

```diff
@@ -1,4 +1,4 @@
-<label class="usa-label">
+<label class="usa-label" id="dob-label">
```

Comment:
To support reuse of this component, can we make this ID not specific to date-of-birth, and also guaranteed [unique](https://dequeuniversity.com/rules/axe/3.5/duplicate-id)?

```suggestion
<label class="usa-label" id="memorable-date-label-<%= unique_id %>">
```